documentation/modules/exploit/windows/fileformat/office_word_hta.md

```diff
@@ -1,11 +1,13 @@
+## Introduction
+
 Microsoft Office is an office suite of applications, servers, and services developed by Microsoft. Microsoft Office contains Microsoft Word, Microsoft Excel, Microsoft PowerPoint and so on. They can support OLE data integration and Virtusl Basic for Application scripting langauage. 
```

Comment:
`Virtusl Basic` ? Should be `Visual Basic` ?

`langauage` should be `language`